hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -566,7 +566,6 @@ public SingleTransactionRecordBuilderImpl fileID(@NonNull final FileID fileID) {
      */
     @Override
     public @NonNull SingleTransactionRecordBuilderImpl contractID(@Nullable ContractID contractID) {
```

Comment:
Is this Nullable ?

Reply:
Added a comment explaining that this argument is `@Nullable` to allow more fluent end-of-transaction record building; e.g. [here](https://github.com/hashgraph/hedera-services/blob/07801-fix-first-xtest/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractCreateHandler.java#L58),
```
        context.recordBuilder(ContractCreateRecordBuilder.class)
                .contractCreateResult(outcome.result())
                .contractID(outcome.recipientIdIfCreated())
                .status(outcome.status());
```

Reply:
Thanks Michael